wordpress-mobile/WordPress-FluxC-Android
# Woo: Shipping label packages endpoint

Fixes #1717.

This PR implements the client and models for the shipping label packages (custom & predefined), which will be used in the shipping label creation flow.

**To test:**

Run the **ShippingLabelStoreTest.'get packages'()** test

In the example app:

1. Go to Woo -> Shipping Labels
2. Select a site with the WooCommerce Shipping & Tax plugin installed
3. Tap on the Get packages button
4. Verify that the returned list of custom and predefined packages matches the core (the titles, dimensions and isLatter flags)
